### PR TITLE
Updated What's next section under components page.

### PR DESCRIPTION
**Pull request by T-Lakshmi**
**This PR updates the What's next section under components page with following:**
1. Consolidated the existing points by removing repetitive word "`Learn about `" .
2. Added few extra reference points which were missing previously.

### CODE REVIEW

**Comment by T-Lakshmi** [comment]
This fixes #37881.

@bradtopol, @lavalamp,
    PTAL!

File: content/en/docs/concepts/overview/components.md [hunk 1]
```diff
@@ -125,7 +125,11 @@ saving container logs to a central log store with search/browsing interface.
...
-* Learn about [kube-scheduler](/docs/concepts/scheduling-eviction/kube-scheduler/)
-* Read etcd's official [documentation](https://etcd.io/docs/)
+Learn more about the following:
+   * [Nodes](/docs/concepts/architecture/nodes/) and communication between [Nodes and control plane](/docs/concepts/architecture/control-plane-node-communication/).
+   * [Controllers](/docs/concepts/architecture/controller/) in Kubernetes.
+   * [kube-scheduler](/docs/concepts/scheduling-eviction/kube-scheduler/) is the default scheduler for Kubernetes and runs as part of the control plane.
+   * Read etcd's official [documentation](https://etcd.io/docs/).
+   * Several common [container runtimes](/docs/setup/production-environment/container-runtimes/) in Kubernetes.
+   * [cloud-controller-manager](/docs/concepts/architecture/cloud-controller/) in detail.
+   * [component tools](/docs/reference/command-line-tools-reference/) such as kubelet, kube-apiserver, kube-controller-manager, kube-proxy etc.
```
Comment (lavalamp) [inline content/en/docs/concepts/overview/components.md:135]: "component tools" is a confusing thing to call these, yes they have CLIs but generally users only need to know kubectl commands

Comment (T-Lakshmi) [reply content/en/docs/concepts/overview/components.md:135]: @lavalamp,
Thanks for your review comment.
> "component tools" is a confusing thing to call these, yes they have CLIs but generally users only need to know kubectl commands

In that case, I will remove reference for component tools and add reference link for `kubectl commands`. Is that ok?
WDYT?

Comment (T-Lakshmi) [reply content/en/docs/concepts/overview/components.md:135]: Hi @lavalamp,
Could you please confirm on above.

**Comment by T-Lakshmi** [comment]
Hi @lavalamp,
I have updated PR by applying your suggestions.
Can you please take another look!!
`Preview Page`:- https://deploy-preview-38191--kubernetes-io-main-staging.netlify.app/docs/concepts/overview/components/#what-s-next

**Comment by lavalamp** [comment]
Maybe I'm not the right person to review this. I don't understand the theme of the new list. (the old list wasn't great either.)

**Comment by mengjiao-liu** [comment]
Can you squash these commits to one?

File: content/en/docs/concepts/overview/components.md [hunk 2]
```diff
@@ -125,7 +125,11 @@ saving container logs to a central log store with search/browsing interface.
...
-* Learn about [Controllers](/docs/concepts/architecture/controller/)
-* Learn about [kube-scheduler](/docs/concepts/scheduling-eviction/kube-scheduler/)
-* Read etcd's official [documentation](https://etcd.io/docs/)
+Learn more about the following:
+   * [Nodes](/docs/concepts/architecture/nodes/) and communication between [Nodes and control plane](/docs/concepts/architecture/control-plane-node-communication/).
+   * [Controllers](/docs/concepts/architecture/controller/) in Kubernetes.
+   * [kube-scheduler](/docs/concepts/scheduling-eviction/kube-scheduler/) is the default scheduler for Kubernetes and runs as part of the control plane.
+   * Read etcd's official [documentation](https://etcd.io/docs/).
+   * Several common [container runtimes](/docs/setup/production-environment/container-runtimes/) in Kubernetes.
+   * [Cloud-Controller-Manager](/docs/concepts/architecture/cloud-controller/) in detail.
```
Comment (sftim) [inline content/en/docs/concepts/overview/components.md:134]:
```suggestion
   * Integrating with cloud providers using [cloud-controller-canager](/docs/concepts/architecture/cloud-controller/).
```
?

Comment (T-Lakshmi) [reply content/en/docs/concepts/overview/components.md]: Hi @sftim ,
Applied the suggested changes, PTAL!

**Comment by T-Lakshmi** [comment]
> Can you squash these commits to one?

Hi @mengjiao-liu,
Updated as per your suggestions, could you please take an another look!
**Preview page: -** https://deploy-preview-38191--kubernetes-io-main-staging.netlify.app/docs/concepts/overview/components/

**Comment by T-Lakshmi** [comment]
Hi @mengjiao-liu, @sftim,
I reworked on your comments, could you please take an another look!!

Thanks in Advance.

File: content/en/docs/concepts/overview/components.md [hunk 3]
```diff
@@ -125,7 +125,11 @@ saving container logs to a central log store with search/browsing interface.
 
 ## {{% heading "whatsnext" %}}
 
-* Learn about [Nodes](/docs/concepts/architecture/nodes/)
-* Learn about [Controllers](/docs/concepts/architecture/controller/)
-* Learn about [kube-scheduler](/docs/concepts/scheduling-eviction/kube-scheduler/)
-* Read etcd's official [documentation](https://etcd.io/docs/)
+Learn more about the following:
+   * [Nodes](/docs/concepts/architecture/nodes/) and communication between [Nodes and control plane](/docs/concepts/architecture/control-plane-node-communication/).
```
Comment (mengjiao-liu) [inline content/en/docs/concepts/overview/components.md:129]:
```suggestion
   * [Nodes](/docs/concepts/architecture/nodes/) and [communication between Nodes and control plane](/docs/concepts/architecture/control-plane-node-communication/).
  
```
Why not select "communication between Nodes and control plane]" in the square brackets as the link? The title that this link points to is "Communication between Nodes and the Control Plane"

**Comment by T-Lakshmi** [comment]
Hi @mengjiao-liu,
Agree on your point, its good to provide reference link name same as page name to avoid confusion.
Committed your suggestion, PTAL!

File: content/en/docs/concepts/overview/components.md [hunk 4]
```diff
@@ -125,7 +125,12 @@ saving container logs to a central log store with search/browsing interface.
...
-* Read etcd's official [documentation](https://etcd.io/docs/)
+Learn more about the following:
+   * [Nodes](/docs/concepts/architecture/nodes/) and [communication between Nodes and control plane](/docs/concepts/architecture/control-plane-node-communication/).
+  
+   * [Controllers](/docs/concepts/architecture/controller/) in Kubernetes.
+   * [kube-scheduler](/docs/concepts/scheduling-eviction/kube-scheduler/) is the default scheduler for Kubernetes and runs as part of the control plane.
+   * Read etcd's official [documentation](https://etcd.io/docs/).
+   * Several common [container runtimes](/docs/setup/production-environment/container-runtimes/) in Kubernetes.
+   * Integrating with cloud providers using [cloud-controller-manager](/docs/concepts/architecture/cloud-controller/).
+   * [kubectl](/docs/reference/generated/kubectl/kubectl-commands) commands.
```
Comment (tengqm) [inline content/en/docs/concepts/overview/components.md:136]: The indentations for the above list items do not make sense.

The revise paragraph doesn't flow well ... some items are phrases, some items are sentences ...

Comment (T-Lakshmi) [reply content/en/docs/concepts/overview/components.md:136]: Hi @tengqm,
Thanks for your review comment,
How about this?

Learn more about the following:
- [Nodes](/docs/concepts/architecture/nodes/) and [their communication](/docs/concepts/architecture/control-plane-node-communication/) with the control plane.
-  Kubernetes [controllers](/docs/concepts/architecture/controller/).
-  [kube-scheduler](/docs/concepts/scheduling-eviction/kube-scheduler/) which is the default scheduler for Kubernetes.
- Etcd’s official [documentation](https://etcd.io/docs/).
- Several [container runtimes](/docs/setup/production-environment/container-runtimes/) in kubernetes.
- Integrating with cloud providers using [cloud-controller-manager](/docs/concepts/architecture/cloud-controller/).
- [kubectl](/docs/reference/generated/kubectl/kubectl-commands) commands.

Comment (T-Lakshmi) [reply content/en/docs/concepts/overview/components.md:136]: Hi @tengqm, @sftim, 
could you please take a look on above preview text and share your feedback.

Comment (T-Lakshmi) [reply content/en/docs/concepts/overview/components.md]: Hi @tengqm,
Updated as per the suggested  feedback, could you please take an another look.

**Comment by T-Lakshmi** [comment]
Deploy preview for the changes: https://deploy-preview-38191--kubernetes-io-main-staging.netlify.app/docs/concepts/overview/components/#what-s-next

File: content/en/docs/concepts/overview/components.md [hunk 5]
```diff
@@ -125,7 +125,11 @@ saving container logs to a central log store with search/browsing interface.
...
 ## {{% heading "whatsnext" %}}
 
-* Learn about [Nodes](/docs/concepts/architecture/nodes/)
-* Learn about [Controllers](/docs/concepts/architecture/controller/)
-* Learn about [kube-scheduler](/docs/concepts/scheduling-eviction/kube-scheduler/)
-* Read etcd's official [documentation](https://etcd.io/docs/)
+Learn more about the following:
+   * [Nodes](/docs/concepts/architecture/nodes/) and [their communication](/docs/concepts/architecture/control-plane-node-communication/) with the control plane.
+   * Kubernetes [controllers](/docs/concepts/architecture/controller/).
+   * [kube-scheduler](/docs/concepts/scheduling-eviction/kube-scheduler/) which is the default scheduler for kubernetes.
```
Comment (mengjiao-liu) [inline content/en/docs/concepts/overview/components.md:131]:
```suggestion
   * [kube-scheduler](/docs/concepts/scheduling-eviction/kube-scheduler/) which is the default scheduler for Kubernetes.
```
Why is Kubernetes lowercase here?

File: content/en/docs/concepts/overview/components.md [hunk 6]
```diff
@@ -125,7 +125,11 @@ saving container logs to a central log store with search/browsing interface.
...
+   * Etcd's official [documentation](https://etcd.io/docs/).
+   * Several [container runtimes](/docs/setup/production-environment/container-runtimes/) in kubernetes.
```
Comment (mengjiao-liu) [inline content/en/docs/concepts/overview/components.md:133]: same as above.

Comment (T-Lakshmi) [reply content/en/docs/concepts/overview/components.md]: Hi @mengjiao-liu,
Updated the case, Thank you.

**Comment by T-Lakshmi** [comment]
Hi @tengqm, @mengjiao-liu, @sftim,
Can you please take a look in this this.
Thanks,

**Comment by T-Lakshmi** [comment]
Hi,
Can anyone please take a look into this PR as it is pending since many days onwards,
Thanks.

**Comment by mengjiao-liu** [comment]
/lgtm

ping @sftim @tengqm for confirmation.

**Comment by T-Lakshmi** [comment]
@mengjiao-liu,
Thanks for providing lgtm.

@tengqm, @sftim,

> ping @sftim @tengqm for confirmation.

Can you please confirm and approve PR, so that it will get merge.

**Pull request merged**